tree/dataframe/test/dataframe_entrylist.cxx

```diff
@@ -33,13 +49,14 @@ void TestTreeWithEntryList()
    auto t = f.Get<TTree>(treename);
    t->SetEntryList(&elist);
 
+   RMTRAII gomt(isMT);
```

Comment:
can this constructor be moved before the creation of the TTree? Commonly, TTree and RDF will have the same IMT status (both on or both off), and I'm not sure it's the case here.